Composer/packages/extensions/adaptive-form/src/utils/resolveBaseSchema.ts

```diff
@@ -6,7 +6,8 @@ import { MicrosoftAdaptiveDialog } from '@bfc/shared';
 import { resolveRef } from './resolveRef';
 
 export function resolveBaseSchema(schema: JSONSchema7, data: MicrosoftAdaptiveDialog): JSONSchema7 | undefined {
-  if (schema.definitions && schema.definitions?.[data.$kind] && schema.definitions?.[data.$kind] !== true) {
-    return resolveRef(schema.definitions?.[data.$kind], schema.definitions);
+  const defSchema = schema.definitions?.[data.$kind];
+  if (defSchema && typeof defSchema === 'object') {
+    return resolveRef(defSchema, schema.definitions);
```

Comment:
Nice - much cleaner.

Reply:
I updated this because typescript started complaining. I'm not really sure why though.